CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
-NEXT
+0.20.0 /
 ====================
+- **BREAKING:** `Apify.utils.requestAsBrowser()` no longer aborts request on status code 406
+  or when other than `text/html` type is received. Use `options.abortFunction` if you want to
+  retain this functionality.
+- **BREAKING:** Added `useLenientHttpParser` option to `Apify.utils.requestAsBrowser()` which
```

Comment:
Wouldn't it be better to call this `useInsecureHttpParser` to match the Node option? Why invent new name for a known thing?

Reply:
Yeah, now with the docs and everything, you're right. I didn't want it to sound too dangerous.

Reply:
Or maybe just `insecureHTTPParser` as Node option is called

Reply:
Actually, that breaks our camel-case naming convention, `useInsecureHttpParser` is better